Tweaking the definition of "enterprise" further. (#33)
openid/ipsie: ipsie-terminology.md

Enterprises are generally defined as entities - e.g. corportations, non-profit organizations, partnerships.  Enterprises have a workforce comprised of employees, contractors, volunteers, and others who operate on behalf of the enterprise.  Enterprises deploy applications and services to support their organizational needs.  Government, non-governmental organizations, educational entities, small businesses, and others may consider themselves enterprises.

> Ultimately, the goal of the IPSIE standard is to better serve the enterprise company. Enterprises prefer to centralize the authentication process of all the applications used across the enterprise. Among other benefits, this allows users to have only one set of credentials to manage, and enables the company to manage which users can access which applications in a central location. In this framing, enterprises are the resource owners where the resources include, but are not limited to, the user directories, the identity providers, the relying parties, the data, and the policies that are used to grant, suspend, and revoke user access.

Reviewer: Here is a suggested definition:

An enterprise refers to an organization that seeks to implement enterprise-grade governance of its users, including authentication, lifecycle management, and risk mitigation. It emphasizes the scale and complexity of managing user access and authentication processes across all applications used within the organization. By centralizing these processes and utilizing an industry-standard set of protocols, the enterprise ensures secure management of user credentials and access, allowing users to manage a single set of credentials while the organization can centrally control and monitor access to various applications.

Reviewer 2: Travis will add the paragraph above to the meeting minutes to further refine

> Ultimately, the goal of the IPSIE standard is to better serve the enterprise company. Enterprises prefer to centralize the authentication process of all the applications used across the enterprise. Among other benefits, this allows users to have only one set of credentials to manage, and enables the company to manage which users can access which applications in a central location. In this framing, enterprises are the resource owners where the resources include, but are not limited to, the user directories, the identity providers, the relying parties, the data, and the policies that are used to grant, suspend, and revoke user access.

Reviewer 2: ```suggestion
Ultimately, the goal of the IPSIE standard is to better serve the enterprise organization. Enterprises prefer to centralize the authentication process of all the applications used across the enterprise. Among other benefits, this allows users to have only one set of credentials to manage, and enables the company to manage which users can access which applications in a central location. In this framing, enterprises have user directories, identity providers, relying parties, data, and the policies that are used to grant, suspend, and revoke user access.
```
